packages/@webex/plugin-meetings/test/unit/spec/multistream/remoteMediaGroup.ts

```diff
@@ -97,6 +97,68 @@ describe('RemoteMediaGroup', () => {
     });
   });
 
+  describe('setPreferLiveVideo', () => {
+    it('sets to true initial option was false', () => {
+
+      const group = new RemoteMediaGroup(fakeMediaRequestManager, fakeReceiveSlots, 255, true, {
+        resolution: 'medium',
+        preferLiveVideo: false,
+      });
+
+      group.setPreferLiveVideo(true);
+
+      assert.calledOnce(fakeMediaRequestManager.cancelRequest);
+
+      assert.calledTwice(fakeMediaRequestManager.addRequest);
```

Comment:
why twice? I guess the first one is from the constructor call in line 103. It would be better to call `fakeMediaRequestManager.addRequest.resetHistory()` before calling `setPreferLiveVideo()` so that here we would expect only 1 call - that would make the test more clear and precise.